tests/test_manager.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for manager."""
+from unittest import TestCase
+from ecosystem.entities import Repository
+from ecosystem.manager import Manager
+
+
+class TestManager(TestCase):
+    """Test class for manager functions."""
+
+    def setUp(self) -> None:
+        with open("ressources/issue.md", "r") as issue_body_file:
+            self.issue_body = issue_body_file.read()
+
+    def test_issue_parsing(self):
+        """"Tests issue parsing function"""
+        self.setUp()
```

Comment:
`setUp` is called automatically before running tests in test class and `tearDown` is called after. You can use them to create resources for tests and clean them after if you need it.

So, you do not need to call them explicitly

Reply:
Done